vochain/state.go

```diff
@@ -481,6 +501,40 @@ func (v *State) IsOracle(addr common.Address) (bool, error) {
 	}(), nil
 }
 
+// SetTreasurer saves the Treasurer address to the state
+func (v *State) SetTreasurer(address common.Address) error {
```

Comment:
Do we really need to export this method?

Reply:
True that.